examples/coreutils/Kraftfile.cat

```diff
@@ -0,0 +1,9 @@
+spec: v0.6
+
+name: cat 
+
+rootfs: ./Dockerfile
+
+cmd: ["/usr/bin/cat"]
```

Comment:
Update the command to list the contents of a file. Add some files to the local directory in the repository than copy them to the `Dockerfile`:

```Dockerfile
COPY myfs/ .
```